pkg/config/config.go

```diff
@@ -174,8 +176,14 @@ func GrantedConfigFolder() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	// check if the .granted folder already exists
-	return path.Join(home, build.ConfigFolderName), nil
+
+	configDir := filepath.Join(home, build.ConfigFolderName)
+	if xdgConfigDir := os.Getenv("XDG_CONFIG_HOME"); !pathExists(configDir) && xdgConfigDir != "" {
+		// ? Should this be "build.ConfigFolderName" or "granted". Directory names in XDG_CONFIG_HOME usually omit the dot prefix. 
+		configDir = filepath.Join(xdgConfigDir, build.ConfigFolderName)
```

Comment:
Let's use just "granted" here. `build.ConfigFolderName` is currently used internally by our core team to run development builds of Granted using `~/.dgranted` as the config folder.

```suggestion
		configDir = filepath.Join(xdgConfigDir, "granted")
```

Reply:
New commit pushed with recommended change.